rules/falco_rules.yaml

```diff
@@ -3397,6 +3397,15 @@
   output: "File execution detected from /dev/shm (proc.cmdline=%proc.cmdline connection=%fd.name user.name=%user.name user.loginuid=%user.loginuid container.id=%container.id evt.type=%evt.type evt.res=%evt.res proc.pid=%proc.pid proc.cwd=%proc.cwd proc.ppid=%proc.ppid proc.pcmdline=%proc.pcmdline proc.sid=%proc.sid proc.exepath=%proc.exepath user.uid=%user.uid user.loginname=%user.loginname group.gid=%group.gid group.name=%group.name container.name=%container.name image=%container.image.repository)"
   priority: WARNING
   tags: [host, container, mitre_execution, mitre_defense_evasion, T1036.005, T1059.004]
+  
+- rule: Get Information About Open Application Windows
+  desc: Detects attempts to get information about open application windows
+  condition: > 
```

Comment:
As best practices we should add the evt.type at the beginning if possibile so in this case you can add `spawned_process` macro. 
In addition I would move the process names in a proper list so the condition would be easily readable.

Reply:
Hi darryk10,
Thank you, for your valuable review.
I have addressed the review and pushed a new commit.